### ISSUE TITLE
Include Certification and Informing PwD of the accessibility of product in Conformance Prototype

### ISSUE THREAD

**Issue by jspellman** [w3c/silver#36]
This issue is an attempt to capture a discussion in [Issue 19](https://github.com/w3c/silver/issues/19) that branched into Conformance Prototype.

**Comment by jspellman** [comment]
From @mbgower: 

In my mind, I had been thinking that a standard that aims to be significantly broader in scope, would likely need to invert its problem statement. For instance, with the fragmentation of UI into any number of customized modalities and restricted use cases, doesn't it make sense to define what any given delivery system can do?

    We need guidelines to indicate the capabilities of any specific user interface
    We need guidelines to indicate the totality of a device's capabilities

In example, a refrigerator or other major appliance may have a reduced-function UI mounted on the door. That UI may be output-only (with inputs coming from fridge activity and environment), or it may have a few user input mechanisms. There may also be manual controls inside the fridge that allow the operator to control some functions (such as temperature or humidity) in response to information revealed on that UI. Obviously the operator can manually add and remove things from the fridge, possibly in response to prompts (i.e., if the UI tells the operator they are out of milk). But the degree to which some such UI information is necessary to the operation of the fridge is debatable, as is whether it should be governed by Silver. Likely there will be a bunch of IoT information, some of which will be available via a web or mobile application, and some of which may be more or less important to the operation of the fridge.

There are a lot of use cases within that, based on user need, user ability, appliance features, etc. There are also considerations for the device itself, which if we are extending considerations beyond the web, likely need to come into play. If I'm in a wheelchair, I may care about door's arc, the reach height for its shelving, or the extension length of the bottom freezer door.

To what extent does the form factor of a device enter into considerations for its 'online' accessibility? In example, with WCAG we were able to more or less assume an external keyboard and screen. Can we assume screens? Input pads? Does a feature-rich app with a reduced voice-controlled interaction constitute a failure? Does an app with an enhanced voice-controlled interaction constitute a failure? (i.e., is there always a need to have equivalent facilitation in every modality for something to be considered to pass Silver?) Does the fact a kiosk's controls may not be within the reach of someone in a wheelchair need to enter into Silver's considerations?

**Comment by jspellman** [comment]
From @mbgower comments in [Issue 19](https://github.com/w3c/silver/issues/19) thread: 


    I'm having a hard time visualizing what you mean by "indicate".

At the moment, the accessibility conformance report (aka VPAT) is not very user friendly and does not allow anyone to easily identify whether a product will be usable by a specific individual (unless it is a completely clean report).

So I was envisioning more focus on results that easily (and hopefully intuitively) wrap up to easily represent the 'passes' from a user perspective. One possible example would be a stamps that validate a product for established disability types -- possibly indicating degrees of usability (sorta like the A - AAA levels).

I realize that WCAG does not concern itself much with reporting, and I'm suggesting that if Silver takes considerations beyond enterprise reports (VPATs), the actual measuring and testing processes may be rejigged to lead to more comprehensible results from a consumer perspective.

This is coming from the reality that relatively few web pages, let alone products, are 100% accessible. So I'm suggesting that understanding if I as a user with, say, dyscalculia can complete the app's primary task would be highly valuable information.

    What is the difference between the capabilities of the user interface, and the totality of the device capabilities?

As per my example, when you begin to address physical objects (i.e., something beyond standard form factor computers). there way well be considerations for direct interaction with the object as well as between the various UIs available for an object. A physical device is likely going to have limitations in design for some users. I as a user may be interested in how much I would be able to interact with the primary modality of a device, and when and to what degree I'm going to have to rely on a different UI (mobile app, web page, dedicated remote) to complete certain tasks.

**Comment by jspellman** [comment]
Thank you for your comment. We have closed this issue because we believe this issue has been addressed or is no longer relevant.  If you believe this issue should be reopened, please open a new issue and reference this issue.
